src/Symfony/Component/Notifier/Bridge/MicrosoftTeams/README.md

```diff
@@ -0,0 +1,23 @@
+Microsoft Teams Notifier
+========================
+
+Provides [Microsoft Teams](https://www.microsoft.com/en/microsoft-365/microsoft-teams/free) integration through Incoming Webhook for Symfony Notifier.
+
+DSN example
+-----------
+
+```
+MICROSOFT_TEAMS_DSN=microsoftteams://HOSTPATH
```

Comment:
this example looks hard to read, because you assume that `HOST` and `PATH` are 2 separate tokens for the explanation but this looks like a single `HOSTPATH` word.